turtle/build.gradle.kts

```diff
@@ -20,6 +20,10 @@ dependencies {
     testImplementation("io.mockk:mockk:1.12.5")
 }
 
+tasks.withType<Test>() {
+    useJUnitPlatform()
+}
```

Comment:
without this configuration, the tests are not run because they are not annotated with Junit 4 @Test annotation